runtime/doc/news.txt

```diff
@@ -84,8 +84,9 @@ The following new APIs or features were added.
 
 • |nvim_set_keymap()| and |nvim_del_keymap()| now support abbreviations.
 
-• Added |lsp-handler| for inlay hints: `textDocument/inlayHint` and
-  `workspace/inlayHint/refresh`
+• Implemented LSP inlay hints - |vim.lsp.buf_inlay_hint_enable()|,
```

Comment:
news.txt (at minimum) is a great place to link to reference URLs such as https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_inlayHint

Reply:
done